routers/api/v1/admin/email.go

```diff
@@ -42,7 +42,7 @@ func GetAllEmails(ctx *context.APIContext) {
 		ListOptions: listOptions,
 	})
 	if err != nil {
-		ctx.Error(http.StatusInternalServerError, "GetAllEmails", err)
+		ctx.APIError(http.StatusInternalServerError, err)
```

Comment:
```suggestion
		ctx.APIErrorInternal(err)
```

Reply:
> This PR is the first step to cleanup the error system.

There are too many, I will leave them to the future.